typescript/forms.d.ts

```diff
@@ -1183,41 +1183,46 @@ export type FormElementLookupStaticDataRecordBase = {
   preFills: FormElementLookupStaticDataPreFill[]
 }
 
-export type FormElementLookupStaticDataRecordText = FormElementLookupStaticDataRecordBase & {
-  inputType: 'TEXT'
-  /**
-   * The value that will be matched exactly on the form element this lookup is
-   * associated when the user is completing the form.
-   */
-  inputValue: string
-}
-export type FormElementLookupStaticDataRecordNumber = FormElementLookupStaticDataRecordBase & {
-  inputType: 'NUMBER'
-  /**
-   * The value that will be matched exactly on the form element this lookup is
-   * associated when the user is completing the form.
-   */
-  inputValue: number
-}
+export type FormElementLookupStaticDataRecordText =
+  FormElementLookupStaticDataRecordBase & {
+    inputType: 'TEXT'
+    /**
+     * The value that will be matched exactly on the form element this lookup is
+     * associated when the user is completing the form.
+     */
+    inputValue?: string
+  }
+export type FormElementLookupStaticDataRecordNumber =
+  FormElementLookupStaticDataRecordBase & {
+    inputType: 'NUMBER'
+    /**
+     * The value that will be matched exactly on the form element this lookup is
+     * associated when the user is completing the form.
+     */
+    inputValue?: number
+  }
 
 export type FormElementLookupStaticDataRecord =
   | FormElementLookupStaticDataRecordText
   | FormElementLookupStaticDataRecordNumber
 
-export type FormElementLookupStaticDataEnvironment = FormElementEnvironmentBase & {
-  /**
-   * Array of records, each associated with a "inputValue" that will determine
-   * the prefill data for the configured form elements based on the
-   * "FormElement.name" property.
-   */
-  records: FormElementLookupStaticDataRecord[]
-}
+export type FormElementLookupStaticDataEnvironment =
+  FormElementEnvironmentBase & {
+    /**
+     * Array of records, each associated with a "inputValue" that will determine
+     * the prefill data for the configured form elements based on the
+     * "FormElement.name" property.
+     */
+    records: FormElementLookupStaticDataRecord[]
+  }
 
 export type NewFormElementLookupBase = {
   /** A human readable identifier for the Lookup. */
   name: string
   /** The identifier for the organisation associated with the Lookup. */
   organisationId: string
+  /** Option to run lookup on submission value being undefined */
+  enableLookupForUndefined?: boolean
```

Comment:
enableLookupForUndefined goes on the form element that is configured with the lookup, rather than on the lookup itself